easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.4.4-GCC-11.2.0.eb

```diff
@@ -21,9 +23,22 @@ description = """ Bowtie 2 is an ultrafast and memory-efficient tool for alignin
 toolchain = {'name': 'GCC', 'version': '11.2.0'}
 toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
 
-source_urls = [('https://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
-sources = ['%(namelower)s-%(version)s-source.zip']
-checksums = ['b9fc1d6e5b898f832c590023bcb15e2a7b76e0ffac486800a9897e7cba2175fc']
+source_urls = [
+    ('https://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download'),
+    'https://github.com/simd-everywhere/simde-no-tests/archive/refs/heads',
```

Comment:
@SIB-software please use a specific commit instead of just the tip of the master branch.

for example, this is the url for the latest commit (as of now):
https://github.com/simd-everywhere/simde-no-tests/archive/cbef1c1.tar.gz

you should also rename the file to a separate version for simd-everywhere, corresponding to the commit date, e.g.:
```
_simd_everywhere_version = '20220504'
```

Reply:
done